critiquebrainz/db/users.py

```diff
@@ -5,19 +5,19 @@
 
 def avatar(user):
     """Args:
-        user: user object from db (dict) or username string
+        user: user data (dict or str)
     Returns:
         URL to gravatar image
     """
     url = "https://gravatar.com/avatar/{0}{1}"
+    link = (user, "?d=identicon")
```

Comment:
I don't understand why you need to inject `?d=identicon`. It can just be `?d=identicon&r=pg` everywhere, there's no need to use `format` for it.